docs/src/install/installation.rst

```diff
@@ -15,8 +15,8 @@ To begin, install ROCm for your platform. For installation instructions, refer t
 
 .. tip::
 
-   If using Bash, we recommend you to set ``PATH=/opt/rocm/bin/:$PATH`` in your ``~/.bashrc`` and refresh your shell using ``source ~/.bashrc``.
-   Alternatively, export the path for your current shell session only, using ``export PATH=/opt/rocm/bin/:$PATH``.
+   If using Bash, set ``PATH=/opt/rocm/bin/:$PATH`` in your ``~/.bashrc`` and refresh your shell using ``source ~/.bashrc``.
+   Alternatively, export the path exclusively for your current shell session, using ``export PATH=/opt/rocm/bin/:$PATH``.
```

Comment:
```suggestion
   Alternatively, export the path for your current shell session using ``export PATH=/opt/rocm/bin/:$PATH``.
```

Reply:
See comment above, developers will know that it won't persist across shell sessions.